test/test_structured_2d.jl

```diff
@@ -70,6 +70,39 @@ end
     end
 end
 
+@trixi_testset "elixir_advection_smview.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_smview.jl"),
+                        l2=[
+                            8.311947673083206e-6,
+                            8.311947673068427e-6,
+                        ],
+                        linf=[
+                            6.627000273318195e-5,
+                            6.62700027264096e-5,
+                        ],
+                        coverage_override=(maxiters = 10^5,))
+
+    @testset "analysis_callback(sol) for AnalysisCallbackCoupled" begin
+        errors = analysis_callback(sol)
+        @test errors.l2 ≈ [
+            8.311947673083206e-6,
+            8.311947673068427e-6,
+        ]
+        @test errors.linf ≈ [
+            6.627000273318195e-5,
+            6.62700027264096e-5,
+        ]
```

Comment:
Just to clarify: Do we need this additional check? IIRC, we had this for the other coupled test to make sure that we exercise all functionality of the AnalysiCallbackCoupled. Here, however, I think it would be sufficient to check that the errors are ok (which we already do with the check above).

Reply:
Yes, because this checks something we haven't tested before (that is, the memory consumption of a simulation with a mesh view)

Reply:
OK. I have removed it.